src/workers/personalizedDigestEmail.ts

```diff
@@ -166,6 +192,55 @@ const worker: Worker = {
       return;
     }
 
+    const analyticsQueue = fastq.promise(
+      async (
+        data: {
+          user_id: string;
+          device_id: string;
+          experiment_id: string;
+          variation_id: number;
+        } & AnalyticsEvent,
+      ) => {
+        await sendAnalyticsEvent([data]);
+      },
+      1,
+    );
+
+    const growthbookClient = getUserGrowthBookInstace(user.id, {
+      enableDevMode: process.env.NODE_ENV !== 'production',
+      subscribeToChanges: false,
+      trackingCallback: async (experiment, result) => {
+        analyticsQueue.push({
+          event_name: 'allocation',
+          event_timestamp: new Date(),
+          user_id: user.id,
+          device_id: 'api',
+          experiment_id: experiment.key,
+          variation_id: result.variationId,
```

Comment:
@idoshamun not sure what `device_id` and `event_name` should be. We don't send it from frontend. Let me know.